cuenca/http/client.py

```diff
@@ -72,20 +73,28 @@ def configure(
             self.jwt_token = Jwt.create(self)
 
     def get(
-        self,
-        endpoint: str,
-        params: ClientRequestParams = None,
+        self, endpoint: str, params: ClientRequestParams = None
     ) -> DictStrAny:
-        return self.request('get', endpoint, params=params)
+        return self._request_json('get', endpoint, params=params)
 
     def post(self, endpoint: str, data: DictStrAny) -> DictStrAny:
-        return self.request('post', endpoint, data=data)
+        return self._request_json('post', endpoint, data=data)
 
-    def patch(self, endpoint: str, data: DictStrAny) -> DictStrAny:
-        return self.request('patch', endpoint, data=data)
+    def patch(self, endpoint: str, data: DictStrAny, **kwargs) -> DictStrAny:
+        return self._request_json('patch', endpoint, data=data)
 
     def delete(self, endpoint: str, data: OptionalDict = None) -> DictStrAny:
-        return self.request('delete', endpoint, data=data)
+        return self._request_json('delete', endpoint, data=data)
+
+    def download(
+        self,
+        endpoint: str,
+        headers={},
+    ) -> bytes:
+        return self.request('get', endpoint, headers=headers)
```

Comment:
This is completely unnecessary. Have `Downloadable.download()` call `Client.request()` directly